### ISSUE TITLE
Prevent creeper explosion lingering effects for summoned creepers

### ISSUE THREAD

**Issue by Yuudaari** [Yuudaari/soulus#38]
There is no official way to disable this functionality.

https://github.com/MinecraftForge/MinecraftForge/issues/4724

**Comment by Yuudaari** [comment]
From the linked issue:
```
this is fishy but since Explosion events can be caught by your mod and Explosions have the source as a field (private, so AT or reflect the field) you could clear the potion effects of creepers (specifically you can remove the potion effects with infinite duration, or only ambient, or whatever you want) when they explode. Since the explosion happens before the cloud is produced this should stop the cloud from being created.
```

**Comment by Yuudaari** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Lingering effects are removed from creeper explosions if the creeper was spawned by a Summoner (as those effects last forever, which is passed onto the lingering potion effect).](https://trello.com/c/AxtxWR6D/239-lingering-effects-are-removed-from-creeper-explosions-if-the-creeper-was-spawned-by-a-summoner-as-those-effects-last-forever-whi)

**Comment by Yuudaari** [comment]
Fixed as of [v4.0.0](https://github.com/Yuudaari/soulus/releases/tag/v4.0.0)
